libs/util.js

```diff
@@ -1,45 +1,7 @@
 'use strict';
 
-module.exports.delay = delay => value => new Promise(resolve => setTimeout(resolve, delay, value));
+export const delay = delay => value => new Promise(resolve => setTimeout(resolve, delay, value));
 
-module.exports.pause = ms => module.exports.delay(ms)();
+export const pause = ms => delay(ms)();
 
-module.exports.wait = ms => () => module.exports.delay(ms);
-
-// Can be removed when Node 8 becomes a requirement
-
-// https://github.com/uxitten/polyfill/blob/master/string.polyfill.js
-// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padStart
-if (!String.prototype.padStart) {
```

Comment:
String padding is [supported in all LTS versions of node now](https://node.green/#ES2017-features-String-padding)